presto-main/src/test/java/io/prestosql/sql/planner/optimizations/TestFullOuterJoinWithCoalesce.java

```diff
@@ -39,6 +40,22 @@
     @Test
     public void testFullOuterJoinWithCoalesce()
     {
+        PlanMatchPattern expected = anyTree(
```

Comment:
it seems that we could add more planner test:
* when join produces symbols from only one side
* when some arguments are "constants"